### ISSUE TITLE
Licensing

### ISSUE THREAD

**Issue by fedeallocati** [resibots/limbo#33]
We should decide which license to use, and whether we should put it at the top of all files or not.

**Comment by costashatz** [comment]
The license is [CeCILL](http://www.cecill.info/index.en.html) and I think we should put something like:

```This file is a part of the ERC ResiBots project.
This software is governed by the CeCILL license under French law and abiding by the rules of distribution of free software. You can use, modify and/ or redistribute the software under the terms of the CeCILL license as circulated by CEA, CNRS and INRIA at the following URL "http://www.cecill.info". As a counterpart to the access to the source code and rights to copy, modify and redistribute granted by the license, users are provided only with a limited warranty and the software's author, the holder of the economic rights, and the successive licensors have only limited liability. In this respect, the user's attention is drawn to the risks associated with loading, using, modifying and/or developing or reproducing the software by the user in light of its specific status of free software, that may mean that it is complicated to manipulate, and that also therefore means that it is reserved for developers and experienced professionals having in-depth computer knowledge. Users are therefore encouraged to load and test the software's suitability as regards their requirements in conditions enabling the security of their systems and/or data to be ensured and, more generally, to use and operate it in the same conditions as regards security. The fact that you are presently reading this means that you have had knowledge of the CeCILL license and that you accept its terms.```

on the top of each code file. And also one LICENSE file should exist.

**Comment by costashatz** [comment]
We could also add one LICENSE file in French as we are part of a French organization.

**Comment by jbmouret** [comment]
I agree. There is a script to do so in sferes: https://github.com/sferes2/sferes2/blob/master/scripts/add_license.sh

And the licenses are here:
https://github.com/sferes2/sferes2/blob/master/scripts/

**Comment by costashatz** [comment]
What's the status on this one? Is there any progress? I can do it if there's not any..

**Comment by jbmouret** [comment]
No progress, and, yes, do not hesitate to do it!

**Comment by costashatz** [comment]
Fixed in #156...
